examples/example_params/gail_params.json

```diff
@@ -9,7 +9,8 @@
     "ML": {
         "Settings": {
             "GPUUse": 0,
-            "NormalizeFeatures": true
+            "NormalizeFeatures": true,
+            "SAC":true
```

Comment:
I believe this SAC parameter is a remnant of @marvinlue and can be deleted.

Reply:
Removed it :+1: